portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase

```diff
@@ -31,7 +31,19 @@ definition {
 	@priority = "3"
 	test VerifyErroWhenJQueryEnabled {
 
-		// TODO LPS-155271 VerifyErroWhenJQueryEnabled pending implementation JQuery enabled
+	ApplicationsMenu.gotoPortlet(
+		category = "Configuration",
+		panel = "Control Panel",
+		portlet = "System Settings");
+
+	SystemSettings.gotoConfiguration(
+		configurationCategory = "Third Party",
+		configurationName = "JQuery",
+		configurationScope = "System Scope");
+
+	FormFields.enableCheckbox(fieldName = "Enable JQuery");
+
+	SystemSettings.saveConfiguration();
```

Comment:
Wrap the poshi steps with tasks( ), please see https://github.com/mateusralv/liferay-portal/pull/41/files#diff-3fdc3c718f6e440ff2b4cee5b74080c0c3ecc97076cd214ea19df03cfdd9f16dR101-R118

Ask @abhnerramos if need help with it.

Reply:
ok